### PR TITLE
[FLOPPY_NEW] Fix MsVC builds

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

Similar to #1714.

### CODE REVIEW

File: drivers/storage/floppy_new/floppy.c [hunk 1]
```diff
@@ -2516,10 +2516,11 @@ Return Value:
 
 #ifndef __REACTOS__
             context = ExAllocatePool(NonPagedPoolNx,
+                                     sizeof(COMPLETION_CONTEXT));
```
Comment (HeisSpiter) [inline drivers/storage/floppy_new/floppy.c]: Useless change, revert

File: drivers/storage/floppy_new/floppy.c [hunk 2]
```diff
@@ -3439,10 +3440,11 @@ Return Value:
     //
 #ifndef __REACTOS__
     parameterList = ExAllocatePool(NonPagedPoolNx,
+                                   sizeof(FORMAT_UNIT_PARAMETER_LIST));
```
Comment (HeisSpiter) [inline drivers/storage/floppy_new/floppy.c]: Useless change, revert

File: drivers/storage/floppy_new/floppy.c (hunk 1 above)
Comment (SergeGautherie) [reply drivers/storage/floppy_new/floppy.c]: On AppVeyor, builds fail without this change(s). Does that not happen for you?

Comment (ThFabba) [reply drivers/storage/floppy_new/floppy.c]: The version that doesn't duplicate the line will fail iff ExAllocatePool is a function-style macro. That will be the case if `classpnp.h` is included and `POOL_TAGGING` is defined... which seems to be the case by default. So the change seems to make sense.

**Pull request merged**